lib/awful/widget/prompt.lua

```diff
@@ -59,6 +59,8 @@ end
 -- @tparam[opt="Run: "] string args.prompt Prompt text.
 -- @tparam[opt=`beautiful.prompt_bg` or `beautiful.bg_normal`] color args.bg Prompt background color.
 -- @tparam[opt=`beautiful.prompt_fg` or `beautiful.fg_normal`] color args.fg Prompt foreground color.
+-- @tparam[opt=`awful.completion.shell`] function args.completion_callback
+--   The callback function to call to get completion.
```

Comment:
i think you should also document here callback function arguments

Reply:
How about just pointing to `awful.prompt.run` here?
`See @{awful.prompt.run} for details.` (I'm not sure if the link works like this)